eis/vocabs/readme.md

```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```

Comment:
Publish how?  You mean the travis-ci errors on this PR?  Ignore those, our testing system is failing on unrelated redirects.